corehq/apps/hqadmin/views.py

```diff
@@ -777,7 +777,7 @@ def view_data(case_id, indicators):
             'case': CommCareCase.get(case_id),
         }
 
-    if user or user_case:
+    if (user or user_case) and domain.call_center_config.use_fixtures:
```

Comment:
👍

On 13 October 2015 at 11:52, Noah Carnahan <notifications@github.com> wrote:

> In corehq/apps/hqadmin/views.py
> <https://github.com/dimagi/commcare-hq/pull/8612#discussion_r41847390>:
>
> > @@ -777,7 +777,7 @@ def view_data(case_id, indicators):
> >              'case': CommCareCase.get(case_id),
> >          }
> >
> > -    if user or user_case:
> > +    if (user or user_case) and domain.call_center_config.use_fixtures:
>
> That's true, however I was wondering if it might lead to confusion if
> fixtures are show here even when use_fixtures is false. I've reverted
> this change and added a caveat to the paragraph on the page instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8612/files#r41847390>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa